DISCOVER/06_childcare.md

```diff
@@ -26,7 +26,8 @@
     - [Corporate Kids Events](https://conferencechildcare.com/)
    - [American Child Care, Inc.](http://www.americanchildcare.com/conventions.htm)
    - [Kiddie Corp](http://www.kiddiecorp.com/)
-    - [Urbansitter](https://www.urbansitter.com/)
+    <!-- ACCESS DENIED: This link is only accessible within the United States. 
+    - [Urbansitter](https://www.urbansitter.com/) -->
```

Comment:
Please, make a footnote that the link is only accessible within the US rather than comment out the entire link.